packages/client/src/comp/states/StateForm.tsx

```diff
@@ -50,9 +51,9 @@ const StateFormSwitch: React.FC<SwitchProps> = ({
   }
 }
 
-const methodExplain = (method: ContactMethod): string => {
-  const email = 'We will email the vote-by-mail signup to your local election official and to you.'
-  const fax = 'We will fax the vote-by-mail signup to your local election official and email a copy to you.'
+const methodExplain = (method: ContactMethod, wording: string): string => {
+  const email = `We will email the ${wording} signup to your local election official and to you.`
```

Comment:
Don't pass in wording.  Pass in state and decide what word to use when rendering the actual sentence.